.factory/automation.yml

```diff
@@ -1218,6 +1218,141 @@ build:
         tool/test/stop-core-server.sh
         exit $TEST_SUCCESS
 
+    deploy-crate-snapshot:
```

Comment:
We will descope this for now, let's push it to later until we know CloudSmith can support what we need.